### PR TITLE
Rename the binaries to libcompose-cli

### PR DESCRIPTION
**Pull request by dave-tucker**
Signed-off-by: Dave Tucker <dt@docker.com>

### CODE REVIEW

File: cli/main/main.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ func main() {
 	factory := &dockerApp.ProjectFactory{}
 
 	app := cli.NewApp()
-	app.Name = "docker-compose"
+	app.Name = "libcompose-cli"
 	app.Usage = "Fast, isolated environments using Docker."
```
Comment (aanand) [inline cli/main/main.go]: Maybe this description should also change to something like "Command-line interface to libcompose"

**Comment by aanand** [comment]
LGTM

**Comment by ibuildthecloud** [comment]
Why would we not want this to be called `docker-compose`.  I think the intention is that if possible this could eventually become the default implementation of compose.  I'd be more in favor of calling this `docker-compose-experimental`.

**Comment by bfirsh** [comment]
I have no opinions on the name, but I think the intention behind this is that the libcompose version of docker-compose is not _quite_ compatible with the Python CLI, so tools that assume there's something called docker-compose that behaves in a certain way will break.

@dave-tucker @aanand Is that right?

**Comment by ibuildthecloud** [comment]
@bfirsh You make an excellent point.  I agree it shouldn't be called `docker-compose` as that is a direct conflict.  I think I still prefer something like `docker-compose-experimental` over `libcompose-cli` because it more clearly states to the user what it is.  My concern with libcompose-cli is that people might look at libcompose as some completely new fancy thing and not understand it at all.  If we are experimenting, we need user to try stuff :)

**Comment by ibuildthecloud** [comment]
At the end of the day the name isn't important and we all agree on that it should be renamed.  So I'm cool with this PR but the tests will break now.

https://github.com/docker/libcompose/blob/master/integration/common.go#L35

I'm really hesitant to merge anything until we get CI running.  So maybe we just hold off until I can get that running?  @jfrazelle Halp!  How do we get this hooked up to Jenkins (I can ping you on IRC monday)

**Comment by jfrazelle** [comment]
Just need GordonTheTurtle to have access to the repo, and 2 web hooks I can get you

**Comment by imikushin** [comment]
Maybe just `compose`?

**Comment by ibuildthecloud** [comment]
@dave-tucker Can you rebase?

**Comment by dave-tucker** [comment]
@ibuildthecloud done. integration tests should now pass. also addressed @aanand's feedback. if needed, i'll happily change the name from `libcompose-cli` to whatever - i have no preference.

**Comment by ibuildthecloud** [comment]
LGTM

**Pull request merged**